test/e2e/Dockerfile.vscode

```diff
@@ -1,7 +1,38 @@
-# Defines an image that installs code-server and the Publisher extension to it
+# Image which contains:
+# Multiple Versions of R, Python and Quarto
+# VS Code server with extensions for python, r and quarto
 
-FROM codercom/code-server:39 AS code-server
+FROM e2ebase AS code-server
+
+RUN export TZ=America/New_York
+
+# 
+# Install code-server
+# 
+
+RUN curl -fsSL https://code-server.dev/install.sh | sh
```

Comment:
Isn't there a way to keep using `FROM codercom/code-server:39 AS code-server`? Could you ellaborate a bit more on why it is needed to install and setup code-server by hand?

Reply:
codercom/code-server image uses Debian GNU/Linux 12. I encountered issues trying to install the additional software needed for the enhanced environment, so I followed the instructions available from the folks who provide that image to install the code-server myself upon the ubuntu22 image. This allowed me to simply replicate the supported software and packages between the client environment and the server environment as well, which I saw as a simplification for the testing environment.